fastai/vision/augment.py

```diff
@@ -21,16 +21,31 @@
 class RandTransform(DisplayedTransform):
     "A transform that before_call its state at each `__call__`"
     do,nm,supports,split_idx = True,None,[],0
-    def __init__(self, p=1., nm=None, before_call=None, **kwargs):
+    def __init__(
+        self,
+        p:float=1., # Probability that Transform'd be applied
+        nm:str=None,
+        before_call=None, # Set up function that executed before __call__
```

Comment:
I think the before call behavior is clear from the name. The purpose of this is that it is done before call which is done before encodes. This is a very convenient method to do batch level preocessing before we ever get to `encodes` at the item level. This is also before type dispatch is applied so we have access to all of the data being passed through the pipeline. 

So I think it would be good to highlight it is a function is applied to the entire batch, including what would not normally be accessible by type dispatch.